test/framework/machinepool_helpers.go

```diff
@@ -57,19 +57,19 @@ func GetMachinePoolsByCluster(ctx context.Context, input GetMachinePoolsByCluste
 	return mps
 }
 
-// WaitForMachinePoolNodesToExistInput is the input for WaitForMachinePoolNodesToExist.
-type WaitForMachinePoolNodesToExistInput struct {
+// WaitForMachinePoolNodesInput is the input for WaitForMachinePoolNodes.
```

Comment:
Is there a need to change these method names as part of this change? It would be good to avoid the churn for consumers of this package if we could.

Reply:
no, there's not a real need. The name just doesn't really match in case of scaling to zero. It initially confused me when I started looking at that code, that's why I renamed it. 
I agree though, this creates churn for consumers of this package.